src/main/java/com/cosmeticos/service/OrderService.java

```diff
@@ -225,9 +230,92 @@ public Order update(OrderRequestBody request) {
 			order.setScheduleId(orderRequest.getScheduleId());
 		}
 
+		//AQUI TRATAMOS O STATUS ACCEPTED QUE VAMOS NA SUPERPAY EFETUAR A RESERVA DO VALOR PARA PAGAMENTO
+		if (orderRequest.getStatus() == Order.Status.ACCEPTED) {
+			this.sendPaymentRequest(orderRequest);
+		}
+
+		//AQUI TRATAMOS O STATUS SCHEDULED QUE VAMOS NA SUPERPAY EFETUAR A RESERVA DO VALOR PARA PAGAMENTO
+		if (orderRequest.getStatus() == Order.Status.SCHEDULED) {
+			this.validateScheduledAndsendPaymentRequest(orderRequest);
+		}
+
 		return orderRepository.save(order);
 	}
 
+	private void validateScheduledAndsendPaymentRequest(Order orderRequest) throws Exception {
+
+		int daysToStart = Integer.parseInt(daysToStartPayment);
+
+		//INSTANCIAMOS O CALENDARIO
+		Calendar c = Calendar.getInstance();
+
+		//DATA ATUAL
+		//EX.: 20/08/2017
+		Date now = c.getTime();
+
+		//PEGAMOS A DATA DE INICIO DO AGENDAMENTO DO PEDIDO
+		Date scheduleDateStart = orderRequest.getScheduleId().getScheduleStart();
+
+		//ATRIBUIMOS A DATA DO AGENDAMENTO DO PEDIDO AO CALENDARIO
+		c.setTime(scheduleDateStart);
+
+		//VOLTAMOS N DIAS, DEFINIDO EM PROPRIEDADES, NO CALENDARIO BASEADO NA DATA DO AGENDAMENTO
+		c.add(Calendar.DATE, -daysToStart);
+
+		//DATA DO AGENDAMENTO MENOS N DIAS NO FORMADO DATE. OU SEJA, A DATA QUE DEVE INICIAR AS TENTAVIDAS DE PAGAMENTO
+		Date dateToStartPayment = c.getTime();
+
+		//SE A DATA ATUAL FOR POSTERIOR A DATA QUE DEVE INICIAR AS TENTATIVAS DE RESERVA DO PAGAMENTO, ENVIAMOS PARA PAGAMENTO
+		if (now.after(dateToStartPayment)) {
+			sendPaymentRequest(orderRequest);
+		}
+	}
+
+	private void sendPaymentRequest(Order orderRequest) throws Exception {
+
+		try {
+			Optional<RetornoTransacao> retornoTransacaoSuperpay = paymentController.sendRequest(orderRequest);
+
+			if(retornoTransacaoSuperpay.isPresent()) {
+
+				//VALIDAMOS SE VEIO O STATUS QUE ESPERAMOS
+				//1 = Pago e Capturado | 2 = Pago e não Capturado (O CORRETO SERIA 1, POIS FAREMOS A CAPTURA POSTERIORMENTE)
```

Comment:
Fiz conforme segue abaixo, somente logando WARN, sem lançar exception ou fazer outra coisa:

`//VALIDAMOS SE VEIO O STATUS QUE ESPERAMOS
				//1 = Pago e Capturado | 2 = Pago e não Capturado (O CORRETO SERIA 2, POIS FAREMOS A CAPTURA POSTERIORMENTE)
				if(retornoTransacaoSuperpay.get().getStatusTransacao() == 2 ||
						retornoTransacaoSuperpay.get().getStatusTransacao() == 1) {

					//SE FOR PAGO E CAPTURADO, HOUVE UM ERRO NAS DEFINICOES DA SUPERPAY, MAS FOI FEITO O PAGAMENTO
					if (retornoTransacaoSuperpay.get().getStatusTransacao() == 1) {
						log.warn("Pedido retornou como PAGO E CAPTURADO, mas o correto seria PAGO E 'NÃO' CAPTURADO.");
					}`